x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/rank/textsimilarity/TextSimilarityRankRetrieverBuilder.java

```diff
@@ -52,6 +55,8 @@ public class TextSimilarityRankRetrieverBuilder extends CompoundRetrieverBuilder
     public static final ParseField FAILURES_ALLOWED_FIELD = new ParseField("allow_rerank_failures");
     public static final ParseField SNIPPETS_FIELD = new ParseField("snippets");
     public static final ParseField NUM_SNIPPETS_FIELD = new ParseField("num_snippets");
+    public static final ParseField CHUNK_SIZE_FIELD = new ParseField("chunk_size");
+    public static final ParseField CHUNKING_SETTINGS_FIELD = new ParseField("chunking_settings");
```

Comment:
`chunk_size` is a shortcut for `chunking_settings` with sentence and no overlap right?

Reply:
Correct, hopefully the updated API will be more transparent.